Libraries/CameraRoll/RCTImagePickerManager.m

```diff
@@ -146,6 +148,16 @@ - (void)imagePickerController:(UIImagePickerController *)picker
     height = @(image.size.height);
     width = @(image.size.width);
   }
+  
+  if (isMovie) {
+    PHFetchResult<PHAsset *> *assets = [PHAsset fetchAssetsWithALAssetURLs:@[[info valueForKey:UIImagePickerControllerReferenceURL]] options:nil];
+    if (assets.count > 0) {
+      PHAsset *videoAsset = assets.firstObject;
+      width = @(videoAsset.pixelWidth);
```

Comment:
micro-nit: `height` before `width` as that order is matched above in 148 for consistency.